autosklearn/automl.py

```diff
@@ -1632,7 +1650,7 @@ def score(self, X, y):
         )
 
     def _get_runhistory_models_performance(self):
-        metric = self._metric
+        metric = self._metric if isinstance(self._metric, Scorer) else self._metric[0]
```

Comment:
Shouldn't it be a list of length 1 if there is only a single metric?

Reply:
Not yet, will update that.